libs/crowdfunding/editor/src/lib/crowdfunding-list.tsx

```diff
@@ -0,0 +1,99 @@
+import React, {useMemo, useState} from 'react'
+import {
+  ListViewActions,
+  ListViewContainer,
+  ListViewHeader,
+  createCheckedPermissionComponent,
+  IconButton,
+  TableWrapper,
+  PaddedCell
+} from '@wepublish/ui/editor'
+import {MdAdd, MdDelete} from 'react-icons/md'
+import {useTranslation} from 'react-i18next'
+import {Link} from 'react-router-dom'
+import {Table as RTable, Table} from 'rsuite'
+import {RowDataType} from 'rsuite/esm/Table'
+import {Crowdfunding, getApiClientV2, useCrowdfundingsQuery} from '@wepublish/editor/api-v2'
+import {CrowdfundingDeleteModal} from './crowdfunding-delete-modal'
+
+const {Column, HeaderCell, Cell: RCell} = RTable
+
+function CrowdfundingList() {
+  const {t} = useTranslation()
+
+  const [crowdfundingDelete, setCrowdfundingDelete] = useState<Crowdfunding | undefined>(undefined)
+
+  const client = useMemo(() => getApiClientV2(), [])
```

Comment:
```suggestion
  const client = getApiClientV2()
```

the function getApiClientV2 is already memoized.